Src/PerformanceCollector/Shared/Implementation/PerformanceCollectorEventSource.cs

```diff
@@ -88,6 +88,13 @@ public void CounterReportAsStrippedEvent(string counter, string stripped, string
             this.WriteEvent(8, misconfiguredCountersCount, overallConfiguredCountersCount, e, this.ApplicationName);
         }
 
+        [Event(15, Level = EventLevel.Verbose,
+            Message = @"Performance counters are unavailable when the application is running under IIS Express. Use EnableIISExpressPerformanceCounters element with a value of 'true' within the Performance Collector Module element to override this behavior.")]
+        public void RunningUnderIisExpress(string applicationName = "dummy")
```

Comment:
I would have never dreamt of having a "dummy" or "incorrect" variable, so I think I copied it from somewhere in SDK :)